analyzer/src/funTest/kotlin/PackageManagerFunTest.kt

```diff
@@ -33,8 +33,11 @@ import io.kotest.matchers.should
 import java.io.File
 
 import org.ossreviewtoolkit.model.config.Excludes
+import org.ossreviewtoolkit.model.config.Includes
```

Comment:
Commit message: I think it would be better to not close #10347 yet but add a checklist to it with changes that need to be done after this PR to complete the feature (like the required changes for the reporter).